.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1090,6 +1147,14 @@ dynamodb_Query:
             - description: Find the movies made in the specified year.
               snippet_tags:
                 - dynamodb.rust.movies-movies_in_year
+    Swift:
+      versions:
+        - sdk_version: 1
+          github: swift/example_code/ddb/basics

```

Comment:
This should be swift/example_code/ddb -- point to the service folder, not the scenario.